TestProjects/UniversalGraphicsTest_Lighting/Assets/Test/Editor/MultipleViewGCTest.cs

```diff
@@ -0,0 +1,51 @@
+using NUnit.Framework;
+using UnityEditor;
+using UnityEngine;
+using UnityEngine.Profiling;
+
+[TestFixture]
+public class MultipleViewGCTest : MonoBehaviour
+{
+    Recorder m_gcAllocRecorder;
```

Comment:
Why is MultipleViewGCTest needed?

Reply:
Same as above...

This was some refactoring done by URP team and then it was reverted. The revert PR is merged to my branch to make things work again:
https://github.com/Unity-Technologies/Graphics/pull/4257/commits/5b8fb9220633fffdfcf9189fbec2eb645fe6508c

Reply:
Please clarify, will a future merge of master make these files disappear from the diff?